report-api/report-templates/statement_report.html

```diff
@@ -0,0 +1,507 @@
+              alt="BC registries Logo" class="logo2">
+            <div class="bc-reg-address">
+              <div><strong>BC Registries</strong></div>
+              <div>161 - 4000 Seymour Place</div>
+              <div>Victoria BC V8X 4S8</div>
+              <br>
+              <div>Toll Free:  1-800-663-6102</div>
+              <div>Email:  bcolhelp@gov.bc.ca</div>
+            </div>
+          </div>
+          <div class="r-heading">
+            <div class="r-title">Deposit Account Statement</div>
+            <div class="r-subtitle">
+              MM-DD-YYYY to MM-DD-YYYY
+            </div>
+          </div>
+        </td>
+      </tr>
+      <tr class="org-account-info-row">
+        <td class="org-address-column">
+          {% if account and account.name %}
+          <div class="org-address">
+            <div class="line-height-reg"><strong>{{account.name}}</strong></div>
+            {% if account and account.contact %}
+            <div class="line-height-reg">{{account.contact.street}}</div>
+            <div class="line-height-reg">{{account.contact.streetAdditional}}</div>
+            <div class="line-height-reg">{{account.contact.city}} {{account.contact.region}} {{account.contact.postalCode}}</div>
+            <div class="line-height-reg">{{account.contact.country}}</div>
+            {% endif %}
+          </div>
+          {% endif %}
+        </td>
+        <td class="account-info">
+          <div class="line-height-reg">
+            <div class="account-info-title">Account Name:</div>
+            <div class="account-info-value">My BC Registries Account</div>
+          </div>
+          <div class="line-height-reg">
+            <div class="account-info-title">Account Number:</div>
+            <div class="account-info-value">#12345678</div>
+          </div>
+          <div class="line-height-reg">
+            <div class="account-info-title">Statement Period:</div>
+            <div class="account-info-value">08-28-2028 to 08-30-1994</div>
+          </div>
+          <div class="line-height-reg">
+            <div class="account-info-title">Due Date:</div>
+            <div class="account-info-value">MM-DD-YYYY</div>
+          </div>
+        </td>
+      </tr>
+      <tr>
+        <td colspan="2">
+          <div class="total-amount-due-row">
+            <div class="total-amount-due-div1">Total Amount Due:</div>
+            <div class="total-amount-due-div2">${{total.fees if total.fees else '0.00'}}</div>
+          </div>
+        </td>
+      </tr>
+      <tr>
```

Comment:
Do not display Transactions, table and Totals if the payments array is coming empty.